src/markdown-pages/landing-es.md

```diff
@@ -1,7 +1,7 @@
 ---
 lang: es
 templateKey: landing-page
-slug: landing
+slug: home
```

Comment:
Is there a specific reason for changing the slug?

Reply:
No specific reason.